MiniApp/Classes/MiniApp.swift

```diff
@@ -47,8 +47,12 @@ public class MiniApp: NSObject {
     ///                         to interact with View component of mini app.
     ///         -   Error: Error details if Mini App View creating is failed
     ///   - messageInterface: Protocol implemented by the user that helps to communicate between Mini App and native application
-    public func create(appId: String, completionHandler: @escaping (Result<MiniAppDisplayProtocol, Error>) -> Void, messageInterface: MiniAppMessageProtocol) {
-        return realMiniApp.createMiniApp(appId: appId, completionHandler: completionHandler, messageInterface: messageInterface)
+    public func create(appId: String,
+                       completionHandler: @escaping (Result<MiniAppDisplayProtocol, Error>) -> Void,
+                       messageInterface: MiniAppMessageProtocol,
+                       userInfoProtocol: MiniAppUserInfoProtocol? = nil
```

Comment:
I'm not really sure about this - We will probably need to be adding even more protocols in the future, so this `create` constructor could end up with way too many parameters if we add it here every time there's a new one 😅 

On the Android side, we added a `setUserInfoHandler` method to `MiniAppMessagePrototocol`. Would this be possible and make sense on iOS?

Reply:
We have used Protocol Composition and Protocol extensions to overcome this issue. Now the Host app doesn't need to conform to any new protocols and also Protocol extensions helps the SDK to add a default implementations for the same, which means, the newly added protocols will be optional for the Host app.